app/models/giveaway.rb

```diff
@@ -13,12 +12,16 @@ def choose_winners
                 return 
             else
                 w.item_won = product[:prod_id]
-                winners << w
+                w.save
             end
         end
       end
     end
 
+    def winners 
+        users.where("item_won <> ''")
```

Comment:
Yeah, I had that set up initially but thought that `giveaway.winners` was nicer to work with than `giveaway.users.winners`. Is there a benefit to using scope vs this method?